pages/admin/courses/_id.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="container">
+    <form>
+      <vue-form-generator
+        :schema="schema"
+        :model="model"
+        :options="formOptions"
+      >
+      </vue-form-generator>
+    </form>
+  </div>
+</template>
+ 
+<script>
+import Vue from 'vue'
+const VueFormGenerator = require('vue-form-generator')
```

Comment:
why not `import` ?  Since this is built via webpack, we can just stick with es6 imports for consistency

Reply:
For some reason it wasn't working I couldn't figure out why that's why i used this.